scripts/e2e.sh

```diff
@@ -20,14 +15,15 @@ cd "${CIRCLE_WORKING_DIRECTORY}/e2e"
 sleep 2
 
 # start ngrok in the background and wait for it to start
-./ngrok config add-authtoken $NGROK_AUTH_TOKEN
-./ngrok http 4141 > /tmp/ngrok.log &
+./ngrok config add-authtoken $NGROK_AUTH_TOKEN > /dev/null 2>&1
```

Comment:
Are there any cases where this might fail and having the stderr output would be useful for debugging, rather than being suppressed? 

If we captured the combined output; and only print it if `$?` is not `0` from it; we would get the best of both worlds.

Alternatively could this be moved into the `Setup deps` step as part of the installation and keeping them together (letting GH deal with hiding the output by default when things are successful)

Reply:
same deal as version management, if it failed or does not work for any reason, we will probably debug directly inside the container though.

Reply:
I could just bring some echo statement back so that we know this step does not work (that is why I disabled all the outputs in here)